project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.1.6
```

Comment:
Any reason not to go straight for 1.2.1?

Reply:
Due to this https://github.com/sbt/sbt/issues/4236, I didn't use 1.2.1. Although it didn't stop the built, [it did cause ERROR at Travis CI](https://travis-ci.org/akka/akka-http-quickstart-scala.g8/jobs/414420928#L489).

Reply:
Yeah it's noisy ;). :+1: then!